### PR TITLE
sprint13-205-config-json

### PR DESCRIPTION
**Pull request by rmathew1011**
Resolves #205 config.json doesn't load inside iframe.

### CODE REVIEW

**Review by jeremythuff — APPROVED** [review]

File: projects/wvr-elements/src/lib/shared/utility/bootstrap.utility.ts [hunk 1]
```diff
@@ -2,10 +2,12 @@ import { Type } from '@angular/core';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 import { AppConfig, APP_CONFIG } from '../config';
 
+/** Provides the current script. */
+const componentScript = document.currentScript;
```
Comment (wwelling) [inline projects/wvr-elements/src/lib/shared/utility/bootstrap.utility.ts:6]: I think it makes sense for this to be within obtainConfigPath scope now that we know the previous issue was from angular cli serve.

Comment (rmathew1011) [reply projects/wvr-elements/src/lib/shared/utility/bootstrap.utility.ts]: `componentScript` moved within the scope of `obtainConfigPath`.

Comment (wwelling) [reply projects/wvr-elements/src/lib/shared/utility/bootstrap.utility.ts]: Thanks. Can you fix the indentation?

Comment (rmathew1011) [reply projects/wvr-elements/src/lib/shared/utility/bootstrap.utility.ts]: Tests at local end passing however the build at Travis end is in progress.

**Pull request merged**